test/images/nvidia/gpu_unit_tests/unit_test

```
@@ -1,5 +1,9 @@
 #!/usr/bin/env bash
 
+set -o errexit
+set -o nounset
+set -o pipefail
```

wouldn't this cause it to fail fast on first failure, rather than the whole suite? I thought suite-level failure was already handled with an explicit `exit` at the end if at least one test fails 
https://github.com/aws/aws-k8s-tester/blob/52246bf8ff4f543ccef7cb0b4faadaa9c926004d/test/images/nvidia/gpu_unit_tests/bash_unit#L595-L622

---

it is yea, https://github.com/aws/aws-k8s-tester/blob/52246bf8ff4f543ccef7cb0b4faadaa9c926004d/test/images/nvidia/gpu_unit_tests/bash_unit#L622

`bash_unit` handles its errors correctly, we're just not reacting to it here